README.md

```diff
@@ -760,7 +760,7 @@ strategies:
           "cpu": 50
           "memory": 50
           "pods": 50
```

Comment:
are these 3 indented wrong too?

Reply:
interesting.. doesn't look that way in my IDE

Reply:
@a7i not to nit about this but maybe the necessary indentation will be more readable if you move the `nodefit` param up above the thresholds in this example